generators/app/templates/resource/tasks/register/coveralls.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * CMD: coveralls
+ *
+ * ---------------------------------------------------------------
+ *
+ * Push coveralls to coveralls.io
+ *
+ */
+module.exports = function(grunt) {
+  grunt.registerTask('coveralls', [
```

Comment:
For semantics it might be better to call this `coverage`

Reply:
Done